example.py

```diff
@@ -144,519 +144,520 @@ def EntityFallback(props):
     return DOM.create_element('span', {'class': 'missing-entity'}, props['children'])
 
 
-config = {
-    # `block_map` is a mapping from Draft.js block types to a definition of their HTML representation.
-    # Extend BLOCK_MAP to start with sane defaults, or make your own from scratch.
-    'block_map': dict(BLOCK_MAP, **{
-        # The most basic mapping format, block type to tag name.
-        BLOCK_TYPES.HEADER_TWO: 'h2',
-        # Use a dict to define props on the block.
-        BLOCK_TYPES.HEADER_THREE: {'element': 'h3', 'props': {'class': 'u-text-center'}},
-        # Add a wrapper (and wrapper_props) to wrap adjacent blocks.
-        BLOCK_TYPES.UNORDERED_LIST_ITEM: {
-            'element': 'li',
-            'wrapper': 'ul',
-            'wrapper_props': {'class': 'bullet-list'},
+if __name__ == '__main__':
```

Comment:
Moving most of the code in here so it's not ran when importing parts of the `example.py` file inside `benchmark.py`.